packages/microcosm/test/unit/observable.test.js

```diff
@@ -370,4 +370,27 @@ describe('Observable', function() {
       )
     })
   })
+
+  describe('scheduler error handling', () => {
+    let raise = null
+
+    beforeEach(() => {
+      raise = scheduler()._raise
+      scheduler()._raise = jest.fn()
```

Comment:
I don't like patching the scheduler as a global singleton (each window gets a unique instance). I wonder how I could write this such that I could control what scheduler the Observable uses.

Reply:
Maybe something like `new Observable(callback, scheduler)`